perfkitbenchmarker/packages/fio.py

```diff
@@ -82,22 +66,51 @@ def ParseJobFile(job_file):
     A dictionary of dictionaries of sample metadata, using test name as keys,
         dictionaries of sample metadata as value.
   """
-  parameter_metadata = {}
+  job_file = job_file.replace(JOB_STONEWALL_PARAMETER, '')
+  config = ConfigParser.ConfigParser()
```

Comment:
I suspect you may want a `RawConfigParser` to prevent interpolation, and to set `allow_no_value=True`.